src/providers/completer/command.ts

```diff
@@ -281,11 +281,9 @@ export class Command implements IProvider {
                             if (node.name === 'documentclass') {
                                 pkg = 'class-' + pkg
                             }
-                            const pkgs = this.extension.manager.cachedContent[file].element.package
-                            if (pkgs) {
+                            const pkgs = this.extension.manager.cachedContent[file].element.package as string[]
```

Comment:
We should not use `as string[]`.  We should check the value even if we know it is not `undefined` now. In the future refactoring, it would be unintentionally `undefined`. And, type checks exist for such unintentional bugs.

Reply:
I was almost sure you were going to say so...

Reply:
The funny thing is that DeepScan detects the new test as an always true condition. Do you know if there is a way to tell DeepScan to ignore it?